test/csuite/timestamp_abort/main.c

```diff
@@ -71,7 +71,7 @@ static char home[1024]; /* Program working dir */
 #define BACKUP_BASE "backup."
 #define CHECK_BASE "check."
 #define INVALID_KEY UINT64_MAX
-#define MAX_BACKUP_INVL 5 /* Maximum interval between backups */
+#define MAX_BACKUP_INVL 4 /* Maximum interval between backups */
 #define MAX_CKPT_INVL 5   /* Maximum interval between checkpoints */
```

Comment:
Up to you about what to do about this. There is already a 1 in 5 chance that there will be 0 delay between backups; this will decrease it to 1 in 4. I'm happy as long as we have enough coverage for different scenarios, both backup having work to do and not having work to do, so that we can have good test coverage.

Reply:
Sounds good.